admin/react/assets/src/dashboard-app/pages/fonts/edit/EditGoogleFont.js

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from "react";
+import { __ } from "@wordpress/i18n";
+import apiFetch from '@wordpress/api-fetch';
+import { useSelector } from 'react-redux';
+
+const EditGoogleVariationItem = ({
+	id,
+	variation
+}) => {
+	const getFontWeightTitle = ( weight ) => {
+		switch ( weight ) {
```

Comment:
I have noticed multiple places this kind of a function is used. So rather then repeating it we can just make a common function and reuse this function @imnavanath , Please check other places too

Reply:
Added on board - https://brainstormforce.atlassian.net/browse/CFP-40 assigned to Babu